### PR TITLE
Removed updateDoi method call inside doi issue method

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: src/Pelagos/Bundle/AppBundle/Rabbit/Consumer/DoiConsumer.php [hunk 1]
```diff
@@ -137,42 +137,41 @@ protected function issueDoi(Dataset $dataset, array $loggingContext)
...
-                $dataset->getTitle(),
-                $dataset->getReferenceDateYear(),
-                'Harte Research Institute'
-            );
-
-            $doi = new DOI($generatedDOI);
-            $doi->setCreator($dataset->getModifier());
-            $doi->setModifier($dataset->getModifier());
-            $dataset->setDoi($doi);
+            $this->logger->info('DOI Already issued for this dataset', $loggingContext);
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Rabbit/Consumer/DoiConsumer.php]: Shouldn't we at least do an update for the DOI?

Comment (praneethpr) [reply src/Pelagos/Bundle/AppBundle/Rabbit/Consumer/DoiConsumer.php]: the last change what i made was exactly that, which led to this bug

File: src/Pelagos/Bundle/AppBundle/Rabbit/Consumer/DoiConsumer.php [hunk 2]
```diff
@@ -137,42 +137,41 @@ protected function issueDoi(Dataset $dataset, array $loggingContext)
...
+                $issueMsg = ConsumerInterface::MSG_REJECT;
+            } catch (HttpServerErrorException $exception) {
+                $this->logger->error('Error requesting DOI: ' . $exception->getMessage(), $loggingContext);
+                //server down. wait for 10 minutes and retry.
+                sleep(self::DELAY_TIME);
+                $issueMsg = ConsumerInterface::MSG_REJECT_REQUEUE;
+            }
         }
-
+
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Rabbit/Consumer/DoiConsumer.php]: extra spaces here.

Comment (praneethpr) [reply src/Pelagos/Bundle/AppBundle/Rabbit/Consumer/DoiConsumer.php]: I'm unable to fix it on phpstorm, if i remove the spaces, the line gets deleted

**Comment by mickel1138** [comment]
I just unlocked an existing DIF, and submitted and approved it again. It issued another DOI.
But the doi issued was a test doi. I'm not sure if we check in our system if the DOI exists. Also this could just be a testing issue.

**Comment by mickel1138** [comment]
Ok, did it another time, and issued another one, again. So doesn't look like a test environment issue.

**Comment by mickel1138** [comment]
So it was my fault, I didn't refresh the consumer. Works as advertised.

File: src/Pelagos/Bundle/AppBundle/Rabbit/Consumer/DoiConsumer.php (hunk 2 above)
Comment (mickel1138) [reply src/Pelagos/Bundle/AppBundle/Rabbit/Consumer/DoiConsumer.php]: vim?

File: src/Pelagos/Bundle/AppBundle/Rabbit/Consumer/DoiConsumer.php (hunk 1 above)
Comment (mickel1138) [reply src/Pelagos/Bundle/AppBundle/Rabbit/Consumer/DoiConsumer.php]: We run update on a separate request after issue I think anyway. So that should work.

**Pull request merged**